.changeset/eleven-fishes-lie.md

```diff
@@ -0,0 +1,5 @@
+---
+"@postenbring/hedwig-tokens": patch
```

Comment:
```suggestion
"@postenbring/hedwig-css": patch
"@postenbring/hedwig-tokens": patch
```

The `tokens.css` is inlined in the css package, so it also needs to be updated for every change in the token package.

💭 We should maybe manage this in a better way